contentcuration/contentcuration/viewsets/websockets/consumers.py

```diff
@@ -174,3 +175,11 @@ def broadcast_success(self, event):
         self.send(text_data=json.dumps({
             'success': success
         }))
+
+# Consumer for handeling the only http request related with
+# Health check
```

Comment:
Why is the comment so far away from the class??? :smile:

Reply:
My auto-save is doing this weirdly!